github/ci/prow/files/jobs/common-templates/common-templates-presubmits.yaml

```diff
@@ -0,0 +1,182 @@
+presubmits:
+  kubevirt/common-templates:
+  - name: pull-rhel6
+    annotations:
+      fork-per-release: "true"
+    always_run: false
+    optional: false
```

Comment:
Can you for now set `optional` to `true`? Otherwise our branchprotector would enforce the test lane as required on every PR.